examples/visualization/plot_study.ipynb

```diff
@@ -0,0 +1,272 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Visualizing High-dimensional Parameter Relationships\n",
+    "\n",
+    "This notebook demonstrates various visualizations of studies in Optuna.\n",
+    "The hyperparameters of a neural network trained to classify images are optimized and the resulting study is then visualized using these features.\n",
+    "\n",
+    "**Note:** If a parameter contains missing values, a trial with missing values is not plotted."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Preparing the Dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "from sklearn.datasets import fetch_openml\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "\n",
+    "mnist = fetch_openml(name='Fashion-MNIST', version=1)\n",
+    "classes = list(set(mnist.target))\n",
+    "\n",
+    "# For demonstrational purpose, only use a subset of the dataset.\n",
+    "n_samples = 4000\n",
+    "data = mnist.data[:n_samples]\n",
+    "target = mnist.target[:n_sample]\n",
```

Comment:
I'm not confident enough but, isn't this a typo?

```suggestion
    "target = mnist.target[:n_samples]\n",
```

Reply:
Sorry I should've verified. You're absolutely right.